tests/io/test_resume_kti.py

```diff
@@ -28,8 +28,8 @@ def test_save_and_load(tmp_path):
     # Check that the files do not exist now
     assert not os.path.isfile(tmp_path / KT_FILENAME)
     assert not os.path.isfile(tmp_path / PROGRESS_FILENAME)
-    resume_kti = create_resume_kti(keys=KEYS,
-                                   texts=TEXTS,
+    parameters = {"keys": KEYS, "texts": TEXTS}
+    resume_kti = create_resume_kti(parameters=parameters,
```

Comment:
Maybe we should modify one of the tests (or add a new one) to include a mask with a different length and make sure everything still works.